src/symbolize/gimli.rs

```diff
@@ -37,6 +37,7 @@ cfg_if::cfg_if! {
         target_os = "haiku",
         target_os = "hurd",
         target_os = "ios",
+        target_os = "visionos",
```

Comment:
`watchos` and `tvos` is missing from here, I think either all of them should be added here or none of them

Reply:
My understanding is that we should, in general, have three cases for relevant cfgs:
- `cfg(all(target_vendor = "apple", target_os = "macos"))`
- `cfg(all(target_vendor = "apple", not(target_os = "macos")))`
- `cfg(target_vendor = "apple")`

I would prefer to see that used, in general.